paddle/fluid/framework/variable.h

```diff
@@ -27,12 +27,12 @@ class Variable {
  public:
   template <typename T>
   const T& Get() const {
-    static_assert(
-        IsRegisteredVarType<T>(),
-        "Not registered type. Please register T inside var_type_traits.h");
-    PADDLE_ENFORCE(holder_ != nullptr, "Variable must hold some thing");
+    static_assert(IsRegisteredVarType<T>(),
+                  "Type T is not registered. Please register T inside "
+                  "var_type_traits.h.");
+    PADDLE_ENFORCE(holder_ != nullptr, "This Variable is not initialized.");
```

Comment:
`This Variable` is also confused to users. Change it to `Variable`.

Reply:
Done